zerver/models.py

```diff
@@ -774,13 +774,15 @@ def authentication_methods_dict(self) -> Dict[str, bool]:
     def __str__(self) -> str:
         return f"<Realm: {self.string_id} {self.id}>"
 
+    # `realm` instead of `self` here to make sure the parameters of the decorated
+    # function match with the original one.
```

Comment:
This comment confused me. It’s actually to make sure the parameters of the *cache key* function match with the original one.